Code/Common/include/sitkImage.h

```diff
@@ -49,7 +49,32 @@ namespace simple
   class PimpleImageBase;
 
   /** \class Image
-   * \brief The main Image class for SimpleITK
+   * \brief The Image class for SimpleITK
+   *
+   * This Image class can represent 2D, 3D, and 4D images. The pixel
+   * types may be a scalar, a multi-component vector or a
+   * run-length-encoded (RLE) "label". The dimension, pixel type and
+   * size is specified at construction and can only be changed by
```

Comment:
Not sure what the phrase "can only be changed by assignment". I would end the sentence "specified at construction." I don't  conceptually think that assigning another image is a change in size or pixel type.